README.md

```diff
@@ -283,9 +283,16 @@ To run the example project, clone the repo, and run `pod install` from the Examp
 
 ## Requirements
 
-* iOS 9.0+
+* iOS 10.0+
 * Swift 5.0
 
+## Installation - Swift Package Manager
+```swift
```

Comment:
Let's add an empty line between the header and the code sample here to be consistent with the rest of the document.

Reply:
Fixed.